src/UI/Implementation/Component/Input/Field/Duration.php

```diff
@@ -35,11 +38,11 @@ public function __construct(
         ilLanguage $lng,
         Factory $field_factory,
         string $label,
-        string $byline
+        ?string $byline
     ) {
         $inputs = [
-            $field_factory->dateTime('start'),
-            $field_factory->dateTime('end')
+            $field_factory->dateTime(self::DEFAULT_START_LABEL),
```

Comment:
@Amstutz is it correct, that the fields have a label 'start', 'end' if we do not pass any "custom-label"? without any translation? I once had a bugreport concernig this (still searching, didn't find it yet)

Reply:
Hmm, no, this certainly isn't. @nhaagen Can you have a look and, in case, provide another PR?